src/routes/mainRouter.js

```diff
@@ -1,13 +1,43 @@
 const express = require("express");
-const { signUp, logIn, logOut } = require("../controllers/userController");
+const {
+  getAllQuiz,
+  insertQuiz,
+  getQuizPerChoice,
+  deleteAllQuiz,
+} = require("../controllers/quizController");
+const {
+  signUp,
+  logIn,
+  getUser,
+  logOut,
+  updateUser,
+  deleteUser,
+} = require("../controllers/userController");
 const signupDataValidate = require("../middlewares/signupDataValidate");
 const loginDataValidate = require("../middlewares/loginDataValidate");
 const authenticateUser = require("../middlewares/authenticateUser");
 const router = express.Router();
 
 //user
+
 router.post("/signup", signupDataValidate, signUp);
-router.route("/login").post(loginDataValidate, logIn).get(authenticateUser);
+router
+  .route("/login")
+  .post(loginDataValidate, logIn)
+  .get(authenticateUser, getUser);
 router.get("/logout", logOut);
+router.put("/updateuser", authenticateUser, updateUser);
+router.delete("/deleteuser", authenticateUser, deleteUser);
+
+//questions
+
+router
+  .route("/quiz")
```

Comment:
Great work setting up the restful endpoint with list retrieval, creation and deletion behavior. I'm used to seeing these types of endpoints named where the list-create-delete endpoint is plural, and the get-update endpoint is singular. For example:

```
/quzzes // list, create, delete many
/quizzes/quiz1 // get, update, delete one
```

What you think of renaming this endpoint?